### DISCUSSION TITLE
Add item to watchlist ??

### DISCUSSION THREAD

**Discussion by jaysonsosweet** [trakt/trakt-api#445] [Q&A] [answered]
Hey guys,

I've been searching everywhere... Where is the damn endpoint to add item to watchlist ??
There's no way you guys didn't developed it, still I can't find it in the doc's Sync section.

Did find that : https://trakt.docs.apiary.io/#reference/sync/add-to-watchlist/add-items-to-watchlist?console=1
in that issue [178](https://github.com/trakt/api-help/issues/178)
but it appears to not working as it redirects to Sync section home page.

Please help.

**Comment by tysonkerridge** [comment, marked as answer]
Try go to the "remove" and scroll up to the one above. Looks like "add" is missing in the sidebar for whatever reason. https://trakt.docs.apiary.io/#reference/sync/remove-from-watchlist/remove-items-from-watchlist

**Reply by jaysonsosweet** [reply]
That's it ! Thx man :)
